lighthouse-core/audits/audit.js

```diff
@@ -167,7 +166,7 @@ class Audit {
       result.rawValue = true;
     }
 
-    if (result.error) {
+    if (result.errorMessage) {
```

Comment:
time to rework re: https://github.com/GoogleChrome/lighthouse/pull/5128#discussion_r186307641  ?